pallets/settlement/src/lib.rs

```diff
@@ -940,6 +958,7 @@ impl<T: Config> Module<T> {
 
         // Check if the venue has required permissions from token owners.
         for ticker in &tickers {
+            // TODO: Merge with above for loop.
```

Comment:
Is there a reason to hold off doing this now?

Reply:
I wanted to check if it was ok to do this kind of cleanup before mainnet.